fourier/fourier_example_test.go

```diff
@@ -0,0 +1,187 @@
+// Copyright ©2018 The Gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package fourier_test
+
+import (
+	"fmt"
+	"math/cmplx"
+
+	"gonum.org/v1/gonum/floats"
+	"gonum.org/v1/gonum/fourier"
+	"gonum.org/v1/gonum/mat"
+)
+
+func ExampleFFT_FFT() {
+	// Period is a set of samples over a given period.
```

Comment:
Period is a set of values observed at regular intervals over a given period.

Reply:
The normal terminology is "sample".